### PR TITLE
[CLI-2851] `kafka region list` output should match `flink region list`

### PR DESCRIPTION
**Pull request by tmalikconfluent**
Release Notes
-------------

Breaking Changes
- Changed column "Region Name" to "Name", "Cloud ID" to "Cloud ", "Region ID" to "Region" and removed "Cloud Name" in the output of `confluent kafka region list`

Checklist
---------
- [x] Leave this box unchecked if features are not yet available in production

What
----
`kafka region list` output should match `flink region list`

References
----------
https://confluentinc.atlassian.net/browse/CLI-2851

Test & Review
-------------
Updated Integration tests.

### CODE REVIEW

**Comment by brianstrauch** [comment]
> kafka region list output columns have been changed.

Can you explain what has changed? i.e. Field "A" has changed to "B".

File: internal/kafka/command_region_list.go [hunk 1]
```diff
@@ -35,7 +43,12 @@ func (c *regionCommand) list(cmd *cobra.Command, _ []string) error {
 
 	list := output.NewList(cmd)
 	for _, region := range regions {
-		list.Add(region)
+		out := &regionOut{
+			Cloud:  strings.ToUpper(region.CloudId),
```
Comment (brianstrauch) [inline internal/kafka/command_region_list.go:47]: Do we have a `nil`-safe `region.GetCloudId()` function available here?

File: test/fixtures/output/kafka/10.golden [hunk 2]
```diff
@@ -1,4 +1,4 @@
-  Cloud ID |     Cloud Name      |   Region ID    |       Region Name        
------------+---------------------+----------------+--------------------------
-  aws      | Amazon Web Services | ap-northeast-1 | ap-northeast-1 (Tokyo)   
-  aws      | Amazon Web Services | us-east-1      | us-east-1 (N. Virginia)  
+           Name           | Cloud |     Region      
+--------------------------+-------+-----------------
+  ap-northeast-1 (Tokyo)  | AWS   | ap-northeast-1
```
Comment (brianstrauch) [inline test/fixtures/output/kafka/10.golden:3]: Previously, "AWS" used to be lowercase. Let's change that back so it matches the backend output.

**Comment by tmalikconfluent** [comment]
> > kafka region list output columns have been changed.
> 
> Can you explain what has changed? i.e. Field "A" has changed to "B".

Almost all the columns have changed to make it similar to flink region list, should i just mention that columns are now similar to flink region list?

File: internal/kafka/command_region_list.go (hunk 1 above)
Comment (tmalikconfluent) [reply internal/kafka/command_region_list.go:47]: we don't have that

File: test/fixtures/output/kafka/10.golden (hunk 2 above)
Comment (tmalikconfluent) [reply test/fixtures/output/kafka/10.golden:3]: does this mean changing it to lower case for all test cases and also in the code?

Comment (brianstrauch) [reply test/fixtures/output/kafka/10.golden:3]: It looks like we're doing `strings.ToUpper(region.CloudId)`, so let's remove the function and update tests accordingly!

File: internal/kafka/command_region_list.go (hunk 1 above)
Comment (brianstrauch) [reply internal/kafka/command_region_list.go:47]: Huh, I wonder why that is. I was assuming the `region` struct is defined in ccloud-sdk-go-v2, but it looks like maybe not?

**Comment by brianstrauch** [comment]
> > > kafka region list output columns have been changed.
> > 
> > 
> > Can you explain what has changed? i.e. Field "A" has changed to "B".
> 
> Almost all the columns have changed to make it similar to flink region list, should i just mention that columns are now similar to flink region list?

I see three changes and one deletion, so how about:

> Renamed field "A" to "B", "C" to "D", "E" to "F", and deleted field "G".

File: internal/kafka/command_region_list.go (hunk 1 above)
Comment (tmalikconfluent) [reply internal/kafka/command_region_list.go:47]: no its not coming directly from  ccloud-sdk-go-v2. The values are in the region.go class of the cli repo.

Comment (brianstrauch) [reply internal/kafka/command_region_list.go]: Ah, ok. And hopefully that struct was initialized with `nil`-safe function calls!

**Review by brianstrauch — APPROVED** [review]
Thank you!!

File: internal/kafka/command_region_list.go (hunk 1 above)
Comment (tmalikconfluent) [reply internal/kafka/command_region_list.go]: yes that is correct!

**Pull request merged**